frontend/app/templates/work_packages.list.new.html

```diff
@@ -0,0 +1,71 @@
+<div cg-busy="vm.loaderPromise" class="work-packages--details-content -create-mode">
+  <div ng-if="vm.workPackage">
+    <h2>{{ ::I18n.t('js.work_packages.create.header') }}</h2>
+
+    <div class="single-attribute subject">
```

Comment:
I'll look into it - to be fair, I have not looked out for code quality in @0xF013's code as this should not be the initial (__main__) criteria here. so this is my fault. 

Am I the only on who is bothered by the fact that create and edit view differ that much?